miio/integrations/roidmi/vacuum/roidmivacuum_miot.py

```diff
@@ -569,7 +568,6 @@ def consumable_status(self) -> RoidmiConsumableStatus:
         return RoidmiConsumableStatus(
             {
                 prop["did"]: prop["value"] if prop["code"] == 0 else None
```

Comment:
```suggestion
                prop["did"]: prop["value"] if prop["code"] == 0 else None
                # max_properties limmit to 10 to avoid "Checksum error" messages from the device.
```